gulpfile.js

```diff
@@ -13,177 +13,179 @@ USWDS SASS GULPFILE
 
 const autoprefixer = require("autoprefixer");
 const csso = require("postcss-csso");
-const gulp = require("gulp");
-const pkg = require("./node_modules/uswds/package.json");
+const { src, dest, series, parallel, watch } = require("gulp");
+const pkg = require("../uswds/package.json").version;
 const postcss = require("gulp-postcss");
 const replace = require("gulp-replace");
 const sass = require("gulp-sass")(require("sass"));
 const sourcemaps = require("gulp-sourcemaps");
-const uswds = require("./node_modules/uswds-gulp/config/uswds");
 const del = require("del");
 const svgSprite = require("gulp-svg-sprite");
 const rename = require("gulp-rename");
+const log = console.log;
+const colorBlue = "\x1b[34m%s\x1b[0m";
```

Comment:
Using [chalk](https://www.npmjs.com/package/chalk) would be preferred, but didn't want to add another dependency.